concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetAccountNonFinalizedTransactions.java

```diff
@@ -0,0 +1,47 @@
+package com.concordium.sdk.examples;
+
+import com.concordium.sdk.ClientV2;
+import com.concordium.sdk.Connection;
+import com.concordium.sdk.Credentials;
+import com.concordium.sdk.exceptions.ClientInitializationException;
+import com.concordium.sdk.requests.BlockHashInput;
+import com.concordium.sdk.requests.getaccountinfo.AccountRequest;
+import com.concordium.sdk.transactions.AccountAddress;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.Callable;
+
+@Command(name = "GetAccountNonFinalizedTransactions", mixinStandardHelpOptions = true)
+public class GetAccountNonFinalizedTransactions implements Callable<Integer> {
```

Comment:
Please add one sentence describing what the example does.

Reply:
done